tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -62,6 +62,8 @@ typedef IEsdlCommand *(*EsdlCommandFactory)(const char *cmdname);
 #define ESDLOPT_XSLT_PATH               "--xslt"
 
 #define ESDLOPT_VERSION                 "--version"
+#define ESDLOPT_INTERFACE_VERSION       "--interface-version"
+#define ESDLOPT_IV                      "-iv"
```

Comment:
"-iv" is considered an abbreviated or short form of "--interface-version" so there is a naming convention to just add an "_S".   Making it #define ESDLOPT_INTERFACE_VERSION_S       "-iv"

Like this example:

#define ECLOPT_INPUT "--input"
#define ECLOPT_INPUT_S "-in"